stdlib/3/os/__init__.pyi

```diff
@@ -121,6 +121,8 @@ if sys.version_info >= (3, 6):
 _PathType = Union[bytes, Text]
 
 if sys.version_info >= (3, 6):
+    _PathType = Union[bytes, Text, PathLike]
```

Comment:
It's not meant to be exported (because it's not in the real stdlib). It's just a shortcut for types used by many definitions in these files.